test/utils/test_skorch.py

```diff
@@ -0,0 +1,198 @@
+from __future__ import annotations
+
+from typing import Any
+
+import pandas as pd
+import pytest
+import torch
+import torch.nn as nn
+from sklearn.datasets import load_diabetes, load_iris
+from sklearn.metrics import accuracy_score, mean_squared_error
+from sklearn.model_selection import train_test_split
+
+from torch_frame import TaskType, stype
+from torch_frame.config.text_embedder import TextEmbedderConfig
+from torch_frame.data.dataset import Dataset
+from torch_frame.data.stats import StatType
+from torch_frame.datasets.fake import FakeDataset
+from torch_frame.nn.models.mlp import MLP
+from torch_frame.testing.text_embedder import HashTextEmbedder
+from torch_frame.utils.skorch import (
+    NeuralNetBinaryClassifierPytorchFrame,
+    NeuralNetClassifierPytorchFrame,
+    NeuralNetPytorchFrame,
+)
+
+
+class EnsureDtypeLoss(nn.Module):
+    def __init__(self, loss: nn.Module, dtype_input: torch.dtype = torch.float,
+                 dtype_target: torch.dtype = torch.float):
+        super().__init__()
+        self.loss = loss
+        self.dtype_input = dtype_input
+        self.dtype_target = dtype_target
+
+    def forward(self, input, target):
+        return self.loss(
+            input.to(dtype=self.dtype_input).squeeze(),
+            target.to(dtype=self.dtype_target).squeeze())
+
+
+@pytest.mark.parametrize('cls', ["mlp"])
+@pytest.mark.parametrize(
+    'stypes',
+    [
+        [stype.numerical],
+        [stype.categorical],
+        # [stype.text_embedded],
+        # [stype.numerical, stype.numerical, stype.text_embedded],
```

Comment:
we don't support these stypes?

Reply:
Currently not supported at this time due to lack of time to understand how to use these dtypes.
However, since it probably only require changes in the arguments of the NeuralNet, it should have little trouble extending it in the future.